js/rpg_core/ImageCache.js

```diff
@@ -0,0 +1,124 @@
+function ImageCache(){
+    this.prototype.initialize.call(this);
```

Comment:
I think `this.prototype` is undefined. (also in RequestQueue)
And, all the other classes use `apply` instead of `call`.
See "Inheritance Style" and other classes one more :-)